authz/audit_logger.go

```diff
@@ -0,0 +1,111 @@
+/*
+ *
+ * Copyright 2023 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package authz
+
+import (
+	"context"
+	"sync"
+)
+
+var (
+	mu sync.Mutex
+	// auditLoggerBuilderRegistry is a map from the name to the audit logger builder.
+	// This backs the RegisterAuditLoggerBuilder API.
```

Comment:
Remove this line? It also backs the GetAuditLoggerBuilder API, and might be used for other things later.

Reply:
Removed.